test/RedisOutputCachingMiddlewareUnitTests/RedisOutputCachingMiddleware.UnitTests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Can we target net6.0 in this one too, for consistency?

Reply:
xunit 2.4.1 does not support net6.0